extensions/amp-analytics/0.1/vendors.js

```diff
@@ -350,6 +350,36 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
     },
   },
 
+  'catchpoint': {
+    'requests': {
+      'endpoint': 'https://r.3gl.net/hawklogserver/r.p',
+      'pageview':'https://r.3gl.net/hawklogserver/r.p?data={"v":2,"y":"load","ui":${UserID},"si":${SessionID},"pi":092934,"di":${AccountID},"pc":1,"dn":${dns},"fc":7,"wt":${wait},"Id":${load},"de":${domInteractive},"dl":${domLoaded},"dc":${docComplete},"rp":1338,"cl":${contentLoad},"dc":${docComplete},"rp":${response},"cl":${contentLoad},"rd":0,"jsc":0,"dh":1200,"dw":1920,"el":193,"rf":"${canonicalUrl}","cv":${isConversion},"rv":${revenue},"ri":${revenue_items}}',
+    },
+    'vars': {
+      'dns': '${navTiming(domainLookupEnd,domainLookupStart)}',
+      'contentLoad': '${navTiming(responseStart,domComplete)}',
+      'docComplete': '${navTiming(navigationStart,loadEventStart)}',
+      'domInteractive': '${navTiming(requestStart,domInteractive)}',
+      'domLoaded': '${navTiming(requestStart,domContentLoadedEventEnd)}',
+      'connect': '${navTiming(connectStart,connectEnd)}',
+      'load': '${navTiming(responseStart,responseEnd)}',
+      'response': '${navTiming(navigationStart,responseEnd)}',
+      'wait': '${navTiming(requestStart,responseStart)}',
+      'ssl': '${navTiming(secureConnectionStart,connectEnd)}',
+      'revenue_items': '',
+      'revenue': '',
+      'isConversion': '',
+      'SessionID': '',
+      'UserID': '',
+    },
+    'triggers': {
+      'trackPageview': {
+        'on': 'visible',
+        'request': 'pageview',
+      },
+    },
+  },
+  
```

Comment:
nit: make sure there's no trailing space this line. It's supposed to be an empty line w/o any space.